go/vt/vttablet/tabletmanager/vreplication/last_error.go

```diff
@@ -48,15 +50,27 @@ func (le *lastError) record(err error) {
 	le.mu.Lock()
 	defer le.mu.Unlock()
 	if err == nil {
+		log.Infof("resetting last error")
```

Comment:
We should include the name here as well so that we can follow the flow in the info log and tie the messages together.